submissions/SpidertronWeaponSwitcher/SpidertronWeaponSwitcher.md

```diff
@@ -4,21 +4,21 @@ author: Xorimuth
 
 ### Under the hood: [Spidertron Weapon Switcher](https://mods.factorio.com/SpidertronWeaponSwitcher)
 
-![Demonstration of switching between weapons](https://github.com/Xorimuth/Alt-F4/blob/master/submissions/SpidertronWeaponSwitcher/SWS-demo-gif.gif?raw=true)
+Spidertron Weapon Switcher is a mod that allows Spidertron to fire any kind of weapon. In this post, I will be talking about how it works behind the scenes. But first, a peak at how awesome this looks in action:
 
-Spidertron Weapon Switcher is a mod that allows a spidertron to fire any weapon. In this post, I will be talking about how it works behind the scenes.
+![Demonstration of switching between weapons](https://github.com/Xorimuth/Alt-F4/blob/master/submissions/SpidertronWeaponSwitcher/SWS-demo-gif.gif?raw=true)
 
-If you’ve never written a mod before, you may not know that many things in Factorio cannot be changed _during the game_, for example, the range of a gun, the graphics of a building and, importantly for me, the weapon slots that a vehicle has. Of course, the weapons that a spidertron has _can_ be changed by mods, but only by creating a new ‘type’ of spidertron during the game’s loading phase.
+If you’ve never written a mod before, you may not know that many things in Factorio cannot be changed _during the game_, like for example the range of a gun, the graphics of a machine and, importantly for me, the weapon slots that a vehicle has. Of course, the weapons that Spidertron has _can_ be changed by mods, but only by creating a new ‘type’ of Spidertron during the loading phase of the game.
 
-The way that I got around this limitation is by creating 5 ‘types’ of spidertron, each one with a different set of weapons. When you switch the weapons of a spidertron, what the mod actually does is:
+The way that I got around this limitation is by creating 5 versions of Spidertron, one for each type of gun. So, when you switch weapons in-game, what the mod actually does is:
```

Comment:
I'd like to keep 'types', because it is directly referencing the other 2 times when I use that word here. Alternatively, change them all for a better word. Of course, I actually mean 'prototype', but I decided to use a less technical-jargony word... there might be a better option.